### ISSUE TITLE
Deletion of entries

### ISSUE THREAD

**Issue by tonyakdewey** [lingdb/ielex2#21]
I don't want to delete things I'm not supposed to.  However, I'm running into quite a few entries that are not, in fact, words.  I'm not sure where the original data collectors got these lexemes, but I've been unable to find them either in dictionaries or in corpus searches.  Can I delete them?  They are all for meanings that have other lexemes that are, in fact, attested, and quite good matches to the Swadesh meaning.

**Comment by cormacanderson** [comment]
Yes, you can delete entries. When on the record of a given lexeme, add "delete" to the url. Please use with care.

**Comment by tonyakdewey** [comment]
I'm not going to delete willy nilly =-).  Only things that are not, in fact, words.
